packages/core-utils/package.json

```diff
@@ -12,6 +12,7 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
```

Comment:
You will have to update the version of `@opentripplanner/types` under `devDependencies`.